extensions/resteasy-reactive/rest-client-reactive-kotlin-serialization/runtime/pom.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>quarkus-rest-client-reactive-kotlin-serialization-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <artifactId>quarkus-rest-client-reactive-kotlin-serialization</artifactId>
+    <name>Quarkus - REST Client Reactive Kotlin Serialization</name>
+    <description>Kotlin serialization support for REST Client Reactive</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-rest-client-reactive</artifactId>
+            <exclusions>
```

Comment:
Are you sure these exclusions are needed? If so, why?

Reply:
This can certainly be discussed but without those deps, RestAssured tries to use jackson serialization, e.g., because it finds those classes and decides that's what it should use.  We could move those exclustions to the ITs to isolate the kotlin serialization functionality.  On the other hand, mixing serialization libraries is a weird thing to do and, arguably, jackson should be an opt in dep at this point.

Reply:
So, it's only needed for test? If so, then please use the proper scope.

Reply:
I've confirmed those exclusions aren't necessary at all and removed them.